### PR TITLE
Add post mortem about absence of notifications from 27th Nov 2023

### PR DESCRIPTION
**Pull request by saraycp**

### CODE REVIEW

File: _posts/deployments/2023-11-30-post-mortem.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+**Resolution**: Initially, we applied a quick patch with [PR #15276](https://github.com/openSUSE/open-build-service/pull/15276) and then we digged into the root cause and fixed it with [PR #15290](https://github.com/openSUSE/open-build-service/pull/15290).
+
+**Detection**: Users warning and error tracking tool.
+
+## Action Items
+
+| Action Item | Owner |
+|---          |---    |
+| Add better notifications alerts. | Notifications opportunity leader. |
```
Comment (danidoni) [inline _posts/deployments/2023-11-30-post-mortem.md:27]: Add an alert in monitoring if no notifications happen in an hour or two?

File: _posts/deployments/2023-11-30-post-mortem.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+| Refactor the subscriptions and notifications codebase, which is quite complex to debug. | Notifications opportunity leader. |
+
+
+## Lessons Learned
+
+**What went well?**
+
+* After the patch was applied, all the notificatios were successfully sent.
```
Comment (danidoni) [inline _posts/deployments/2023-11-30-post-mortem.md:35]:
```suggestion
* After the patch was applied, all the notifications were successfully sent.
```

File: _posts/deployments/2023-11-30-post-mortem.md (hunk 1 above)
Comment (saraycp) [reply _posts/deployments/2023-11-30-post-mortem.md:27]: That's the idea, yes. I rephrased the sentence.

**Comment by saraycp** [comment]
@danidoni @eduardoj thanks for your feedback, all addressed.

**Review by eduardoj — APPROVED** [review]

**Review by eduardoj — DISMISSED** [review]

**Comment by danidoni** [comment]
Why are there two files?

**Comment by saraycp** [comment]
@danidoni @eduardoj sorry for the mess. Now there is only one file with today's date.

I've corrected a few typos in that file and rephrased some sentences in the Timeline section. I'd appreciate you have a quick last look.

**Review by danidoni — APPROVED** [review]

**Pull request merged**